conf/modules/px4io_flash.xml

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="px4io_flash">
+  <doc>
+    <description>Flashes the px4io f1 through the px4 bootloader.</description>
+  </doc>
+  <header>
+    <file name="px4io_flash.h"/>
+  </header>
+  <init fun="px4ioflash_init()"/>
+  <event fun="px4ioflash_event()"/>
+  <makefile target="ap">
+    <raw>
+        PX4IO_UART ?= UART6
```

Comment:
Also Baud rates should be always prefixed with `B`, so `B1500000` instead of `1500000`
That define then needs to be added in `sw/airborne/arch/stm32/mcu_periph/uart_arch.h`